### ISSUE TITLE
Not all URL parameters work in iframe embed player

### ISSUE THREAD

**Issue by davidmpurdy** [B-Interactive/cloudflare-stream-wordpress#14]
It appears that the iframe player treats boolean parameters that are `false` by default as "true" if they are present at all in the querystring (even if they are set to `false`). 

Affected parameters I've identified are `muted`, `loop`, and `autoplay`. (The behavior is [documented for autoplay](https://developers.cloudflare.com/stream/viewing-videos/using-the-stream-player/#basic-options).)

For example, the following iframe src results in a video that **does** autoplay, start muted, and loop. (But does **not** show controls.)

`https://iframe.cloudflarestream.com/uid123456?muted=false&preload=false&loop=false&autoplay=false&controls=false`

I found this updating the block, but it affects shortcodes as well. For example, the following shortcode generates a URL like the one above which treats autoplay, loop, and muted as effectively true. 

```
[cloudflare_stream uid="[uid]" controls="false" autoplay="false" loop="false" preload="false" muted="false"]
```

I've [raised the issue](https://community.cloudflare.com/t/iframe-player-boolean-url-parameters-that-default-to-false-seem-to-go-by-presence-not-value/417184) of at least documenting the behavior, if not allowing false values. 

I think the best solution is to add some logic to where querystring parameters are added in [Cloudflare_Stream_API::get_video_embed](https://github.com/B-Interactive/cloudflare-stream-wordpress/blob/79ff878fff96d8f53122db14c1cf5673889068cf/src/inc/class-cloudflare-stream-api.php#L262) to simply not add parameters that are falsy or `"false"`. 

I'm working on a PR now.

**Comment by B-Interactive** [comment]
Playing with the embed options in the Cloudflare Stream console, it would seem to confirm what you've found, and also for `preload`.  The assumption that specifying the inverse would set it as such, was [an error on my part](https://github.com/B-Interactive/cloudflare-stream-wordpress/commit/f11a1c5be0140d02ddf1836082f59b7da68d52c8#diff-16cfedd3ba19ac75a5ee3ad37e982e1371192f645c7a597eaceea5f6fe79b3a3).

Default when no options specified:
- controls (true)
- autoplay (false)
- muted (false)
- loop (false)
- preload (false)

**Comment by davidmpurdy** [comment]
I didn't include `preload` because in my testing it looks to work a little differently - it appears to pass through to the `<video>` element if the [value is valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-preload), but if I recall seems to work OK with mapping `false` to `none` and `true` to `auto`. 

I meant to raise a separate issue of how to handle it. The [Stream documentation](https://developers.cloudflare.com/stream/viewing-videos/using-the-stream-player/#basic-options) seems to suggest that it wants the HTML enumerated options, so maybe we should add a line to map `true` and `false` values on our end (seems worth keeping them working for backwards compatibility).

**Comment by B-Interactive** [comment]
Fix implemented in 1cb20385ab9c7a9754a29d0b7315b36afadb45f5.

**Comment by davidmpurdy** [comment]
These seem to be broken again for me. At first glance from inspecting the page source and the iframe URL, I'm wondering if it has something to do with [encoded ampersands](https://github.com/B-Interactive/cloudflare-stream-wordpress/commit/1cb20385ab9c7a9754a29d0b7315b36afadb45f5#commitcomment-83912203)...?
